testing/web3signer_tests/tls/lighthouse/cert.pem

```diff
@@ -1,32 +1,33 @@
 -----BEGIN CERTIFICATE-----
```

Comment:
Ah, I also just re-generated these in #4650. You'll get a merge conflict but I don't mind if you just clobber the keys I generated. Explicitly specifying aes256 is probably better than what I did (relying on the default).